### PR TITLE
Enhance/fuku global arguments

### PR DESCRIPTION
**Pull request by alpha1d3d**
- Added `setup.cfg` to bring some consistency with isort and flake8
- flake8 and isort-ed some files
- Added global arguments for `app` and `pg` which overwrite the session context (or is used if no arg in the context)
This allows us to run commands like `fuku --app=XXX service redeploy fg` **without** having to `fuku app sl XXX`

### CODE REVIEW

File: fuku/task.py [hunk 1]
```diff
@@ -2,12 +2,16 @@
 
 from .module import Module
 from .utils import (
-    StoreKeyValuePair, StorePortPair,
```
Comment (furious-luke) [inline fuku/task.py:5]: Hmmm. I guess.

File: fuku/task.py [hunk 2]
```diff
@@ -95,6 +99,7 @@ def add_arguments(self, parser):
         p.set_defaults(task_handler=self.handle_prune)
 
     def handle_list(self, args):
+        print(args)
```
Comment (furious-luke) [inline fuku/task.py]: Should this be here?

File: fuku/task.py [hunk 3]
```diff
@@ -260,7 +265,7 @@ def handle_ports_unset(self, args):
 
     def ports_unset(self, name, values):
         app_task = self.get_app_task()
-        ctr_def = self.get_container_definition(task, name)
+        ctr_def = self.get_container_definition(app_task, name)
```
Comment (furious-luke) [inline fuku/task.py:267]: Ooh, good catch!

File: fuku/task.py (hunk 2 above)
Comment (alpha1d3d) [reply fuku/task.py]: Removed!

File: fuku/task.py (hunk 3 above)
Comment (alpha1d3d) [reply fuku/task.py:267]: Thats what flake8 is for :P

File: fuku/task.py (hunk 1 above)
Comment (alpha1d3d) [reply fuku/task.py:5]: Not used, unless u had plans for it? =/

Comment (furious-luke) [reply fuku/task.py:5]: Not the removal, but putting everything on individual lines. Hmmph. But okay.

**Pull request merged**